src/simplewallet/simplewallet.cpp

```diff
@@ -5070,6 +5064,67 @@ bool simple_wallet::stake(const std::vector<std::string> &args_)
     return true;
   }
 
+  try
+  {
+    const auto& response = m_wallet->get_service_node_list();
+    bool found = false;
+    bool full = false;
+    uint64_t can_contrib_total = 0;
+    uint64_t must_contrib_total = 0;
+    for (const auto& snode_info : response.service_node_states)
```

Comment:
Please add the squiggly braces for multi-line for loops.

Reply:
nice, fixed